apollo-router/src/spec/schema.rs

```diff
@@ -601,14 +593,9 @@ mod tests {
             Err(SchemaError::Api(s)) => {
                 assert_eq!(
                     s,
-                    r#"The supergraph schema failed to produce a valid API schema. Caused by:
-Input field "InputObject.privateField" is @inaccessible but is used in the default value of "@foo(someArg:)", which is in the API schema.
-
-GraphQL request:42:1
-41 |
-42 | input InputObject {
-   | ^
-43 |   someField: String"#
+                    r#"The supergraph schema failed to produce a valid API schema: The following errors occurred:
+
+  - Input field `InputObject.privateField` is @inaccessible but is used in the default value of `@foo(someArg:)`, which is in the API schema."#
```

Comment:
The formatting is a bit worse here but we would follow up on that once QP itself is getting to a usable shape